components/molecules/CardProfile/card-profile.tsx

```diff
@@ -14,27 +17,29 @@ interface CardProfileProps {
 
 const CardProfile = ({ githubAvatar, githubName, totalPRs, dateOfFirstPR }: CardProfileProps): JSX.Element => {
   return (
-    <a href={`https://www.github.com/${githubName}`} target="_blank" className="flex items-center gap-2" rel="noreferrer">
-      <Avatar size={40} avatarURL={githubAvatar ? githubAvatar : undefined} />
-      <div>
+    <Link href={`/user/${githubName}`} className="flex items-center gap-2">
+      <a>
+        <Avatar size={40} avatarURL={githubAvatar ? githubAvatar : undefined} />
         <div>
-          <Text className="!text-base !text-black  ">{githubName}</Text>
-        </div>
-        <div className="flex gap-2   text-xs">
-          <div className="flex text-xs items-center gap-1 text-light-slate-11">
-            {totalPRs !== undefined && (
-              <>
-                <Icon size={12} alt="Total Pull-Requests" IconImage={ForkIcon} /> {totalPRs} PR
-                {totalPRs === 1 ? "" : "s"}
-              </>
-            )}
+          <div>
+            <Text className="!text-base !text-black  ">{githubName}</Text>
```

Comment:
remove the `!` important classes here please and let's debug why you need to use them!

Reply:
I don't know if `!` is needed, but it was introduced in https://github.com/open-sauced/insights/pull/232